jsk_unitree_robot/cross/build_ros1_dependencies.sh

```diff
@@ -4,6 +4,7 @@ TARGET_MACHINE="${TARGET_MACHINE:-arm64v8}"
 HOST_INSTALL_ROOT="${BASE_ROOT:-${PWD}}/"${TARGET_MACHINE}_System
 INSTALL_ROOT=System
 SOURCE_ROOT=${TARGET_MACHINE}_ws_ros1_dependencies_sources
+JSK_UNITREE_COMPILE_PARALLEL="${JSK_UNITREE_COMPILE_PARALLEL:-4}"
```

Comment:
Is this the same functionality as  MAKEFLAGS? https://stackoverflow.com/questions/42291992/how-can-i-specify-the-default-number-of-jobs-to-run-from-in-a-makefile
If we pass MAKEFLAGS, it will passed to catkin? -> https://github.com/ros/catkin/issues/1047

Reply:
Thank you.
I reverted the commits and create new commit to passthrough MAKEFLAGS.
https://github.com/k-okada/jsk_robot/pull/66/commits/c88ebe04b65390e6915bdb8e210b0e4eba948752